packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -32,15 +41,31 @@ interface AutocompleteProps {
    * is retuned from this method to the user as available options.
    * @param newInputText
    */
-  getOptions(newInputText: string): Option[] | Promise<Option[]>;
+  getOptions(
+    newInputText: string,
+  ): OptionCollection | Promise<OptionCollection>;
+
+  /**
+   * Optional additional blur behaviour (clicking away from the input text or
```

Comment:
We don't need to indicate that this is optional in test here as the `?` on the param shows that. The docs viewer will use that to indicate that this is optional.